app/jobs/apply_api/import_applications_job.rb

```diff
@@ -9,7 +9,9 @@ def perform
 
       new_applications.each do |application_data|
         application_record = ImportApplication.call(application_data: application_data)
-        Trainees::CreateFromApply.call(application: application_record)
+        Trainees::CreateFromApply.call(application: application_record) if application_record.present?
+      rescue ApplyApiMissingDataError
+        Sentry.capture_message "The data from Apply has missing attributes"
```

Comment:
Is there something we can add to the message here that will help us identify which application the error is in? The application_id maybe? I think it has something like that.

Reply:
Yes, I'll add the Application ID to the error message 👍